docs/sources/operators-guide/tools/mimirtool.md

```diff
@@ -72,6 +72,14 @@ The following sections outline the commands that you can run against Grafana Mim
 
 The following commands interact with Grafana Mimir Alertmanager configuration and alert template files.
 
+#### Configuration
+
+This configuration applies to all `mimirtool alerts` and `mimirtool alertmanager` commands.
+
+| Environment variable | Flag           | Description                                                                |
+| -------------------- | -------------- | -------------------------------------------------------------------------- |
+| `MIMIR_API_PREFIX`   | `--api-prefix` | Sets the prefix for the Grafana Mimir endpoints. Default is '/prometheus'. |
+
```

Comment:
I left a comment to remove this from the `alertmanager` command. You'll also need to update the docs - remove the mention of alertmanager and move this to the `alerts verify` command